src/integration/common/SEO.js

```diff
@@ -1,151 +1,184 @@
-        );
-
-        expect(fbAdminsEl.getAttribute('content')).toBe(fbAdmins);
-        expect(fbAppIdEl.getAttribute('content')).toBe(fbAppId);
-        expect(fbPagesEl.getAttribute('content')).toBe(fbPages);
-        expect(ogImageEl.getAttribute('content')).toBe(ogImage);
-        expect(ogImageAltEl.getAttribute('content')).toBe(ogImageAlt);
-        expect(ogLocaleEl.getAttribute('content')).toBe(ogLocale);
-        expect(ogTypeEl.getAttribute('content')).toBe(ogType);
-        expect(ogUrlEl.getAttribute('content')).toBe(ogUrl);
-        expect(ogSiteNameEl.getAttribute('content')).toBe(ogSiteName);
-        expect(twitterCardEl.getAttribute('content')).toBe(twitterCard);
-        expect(twitterCreatorEl.getAttribute('content')).toBe(twitterCreator);
-        expect(twitterImageAltEl.getAttribute('content')).toBe(twitterImageAlt);
-        expect(twitterImageSrcEl.getAttribute('content')).toBe(twitterImageSrc);
-        expect(twitterSiteEl.getAttribute('content')).toBe(twitterSite);
-        expect(ogDescriptionEl.getAttribute('content')).toBe(ogDescription);
-        expect(ogTitleEl.getAttribute('content')).toBe(ogTitle);
-        expect(twitterDescriptionEl.getAttribute('content')).toBe(
-          twitterDescription,
-        );
-        expect(twitterTitleEl.getAttribute('content')).toBe(twitterTitle);
-      });
-
-      it('it can see the linked data', () => {
-        expect(
-          page.document.querySelector('script[type="application/ld+json"]')
-            .textContent,
-        ).toEqual(linkedData);
-      });
+const getMetaTagContent = selector =>
+  document.querySelector(selector).getAttribute('content');
+
+export default () => {
+  describe('SEO', () => {
+    it('Page title', () => {
+      const pageTitleEl = document.querySelector('title');
+
+      expect(pageTitleEl).toBeInTheDocument();
+      expect(pageTitleEl.textContent).toBeTruthy();
+      expect(pageTitleEl.textContent).toMatchSnapshot();
+    });
+
+    it('Heading level 1', () => {
+      const headingEl = document.querySelector('h1');
+
+      expect(headingEl).toBeInTheDocument();
+      expect(headingEl.textContent).toBeTruthy();
+      expect(headingEl.textContent).toMatchSnapshot();
+    });
+
+    it('Canonical link', () => {
+      const canonicalEl = document.querySelector('head link[rel="canonical"]');
+
+      expect(canonicalEl.getAttribute('href')).toMatchSnapshot();
+    });
+
+    it('Robots meta tag', () => {
+      const robotsEl = document.querySelector('head meta[name="robots"]');
+      const robotsContent = robotsEl.getAttribute('content');
+
+      expect(robotsContent).toMatchSnapshot();
+    });
+
+    it('Lang attribute', () => {
+      const htmlEl = document.querySelector('html');
+
+      expect(htmlEl.getAttribute('lang')).toMatchSnapshot();
+    });
+
+    it('Dir attribute', () => {
+      const htmlEl = document.querySelector('html');
+
+      expect(htmlEl.getAttribute('dir')).toMatchSnapshot();
+    });
```

Comment:
Do any of these need a `truthy` check for the value of the attribute, like the metatdata content tests below?